packages/sdk/src/actions/getAvailableRoutes.ts

```diff
@@ -7,21 +7,24 @@ export type RoutesQueryParams = Partial<{
   /**
    * The origin token address. If set only routes with this token as origin are returned.
    */
-  originToken: Address;
+  originToken?: Address;
```

Comment:
Are these `?` necessary? Isn't this already covered with `Partial`?

Reply:
Oh I must've not seen that lol